Resources/doc/index.rst

```diff
@@ -50,7 +50,7 @@ The default configuration is as follow:
     .. code-block:: yaml
 
         sensio_framework_extra:
-            router:      { annotations: true }
+            router:      { annotations: true } # Deprecated; use routing annotations of Symfony core instead
```

Comment:
`annotation`?  since there's only one in core

Reply:
It's indefinite. Otherwise it sounds bad